src/ansys/units/quantity.py

```diff
@@ -5,8 +5,15 @@
 
 import ansys.units as ansunits
 
+try:
+    import numpy as np
```

Comment:
We should add numpy as an optional dependency I think if we're having an explicit import like this